packages/openapi-spec-builder/package.json

```diff
@@ -9,7 +9,7 @@
     "build": "lb-tsc es2017",
     "build:apidocs": "lb-apidocs",
     "clean": "lb-clean loopback-openapi-spec-builder*.tgz dist package api-docs",
-    "prepublishOnly": "npm run build && npm run build:apidocs",
+
```

Comment:
Could you please remove these empty lines? They will be eventually removed by `npm` anyways.